layers/dns_test.go

```diff
@@ -576,10 +576,8 @@ var testMalformedRootQuery = []byte{
 
 func TestMalformedRootQuery(t *testing.T) {
 	p := gopacket.NewPacket(testMalformedRootQuery, LinkTypeEthernet, testDecodeOptions)
```

Comment:
Same for the variable containing the packet

Reply:
Changed to testBlankNameRootQuery.